unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Billboard/Handler/BillboardComponentHandler.cs

```diff
@@ -0,0 +1,103 @@
+﻿using DCL.Controllers;
+using DCL.ECSRuntime;
+using DCL.Models;
+using DCL.ECSComponents;
+using UnityEngine;
+
+namespace DCL.ECSComponents
+{
+    public class BillboardComponentHandler : IECSComponentHandler<PBBillboard>
+    {
+        private readonly IUpdateEventHandler updateEventHandler;
+        
+        private Transform entityTransform;
+        private Vector3Variable cameraPosition => CommonScriptableObjects.cameraPosition;
+        private UnityEngine.Vector3 lastPosition;
+
+        private IDCLEntity entity;
+        private IParcelScene scene;
+        private PBBillboard model;
+        
+        public BillboardComponentHandler(IUpdateEventHandler updateEventHandler)
+        {
+            this.updateEventHandler = updateEventHandler;
+            updateEventHandler.AddListener(IUpdateEventHandler.EventType.LateUpdate, LateUpdate);
```

Comment:
We should start discussing moving all components that require an update loop to a more "System" approach in the near future

Reply:
Sure! Right now this is the option that we have but we can start discussing it

Reply:
I agree!